app/Locale/ChineseSimplified.php

```diff
@@ -179,8 +179,6 @@ public function get_locale(): array
 			'ALBUM_VIDEOS' => '视频数',
 			'ALBUM_SUBALBUMS' => '子相册数',
 			'ALBUM_SHARING' => '共享',
-			'ALBUM_SHR_YES' => '是',
-			'ALBUM_SHR_NO' => '否',
```

Comment:
This is used in `Http/Livewire/Sidebar.php`.

Reply:
Indeed.
Fixed [3352064](https://github.com/LycheeOrg/Lychee/pull/1539/commits/33520643cd330bbb7fe366ddbbc9fa8c9dfb364f).